app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -1223,6 +1246,7 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
 
         val showPlaceholder = adapter.itemCount == 0 || isDestroyed
         binding.placeholderText.isVisible = showPlaceholder
+        binding.conversationRecyclerView.visibility = if (showPlaceholder) View.INVISIBLE else View.VISIBLE
```

Comment:
Are you sure you want `View.INVISIBLE` (which takes up space) rather than `View.GONE` which doesn't? Up to you - this may be exactly what you want, but thought I'd double check.

Reply:
Yes, otherwise the layout will be wrong